src/components/Input/README.md

```diff
@@ -54,6 +54,8 @@ Note: Be sure the version for `Button` is at least version 2.
 | ------------------------ | -------------------------- | -------------------------------------------------------------------------- |
 | action                   | `React.Component`          | Embedded actions for the Input.                                            |
 | autoFocus                | `boolean`                  | Automatically focuses the input.                                           |
+| charValidatorLimit       | `number`                   | How many chars are allowed to be input.                                    |
+| charValidatorShowAt      | `boolean`                  | At how many chars input should the charValidator be shown.                 |
```

Comment:
Should this be listed a `boolean` value or `number`?

Reply:
Good catch, thanks, fixed!